edge/cmd/edgecore/edgecore.go

```diff
@@ -1,19 +1,17 @@
 package main
 
 import (
-	"github.com/kubeedge/beehive/pkg/core"
-	"github.com/kubeedge/kubeedge/edge/pkg/common/dbm"
-	_ "github.com/kubeedge/kubeedge/edge/pkg/devicetwin"
-	_ "github.com/kubeedge/kubeedge/edge/pkg/edged"
-	_ "github.com/kubeedge/kubeedge/edge/pkg/edgehub"
-	_ "github.com/kubeedge/kubeedge/edge/pkg/eventbus"
-	_ "github.com/kubeedge/kubeedge/edge/pkg/metamanager"
-	_ "github.com/kubeedge/kubeedge/edge/pkg/servicebus"
+	"os"
+
+	"github.com/kubeedge/kubeedge/edge/cmd/edgecore/app"
 	_ "github.com/kubeedge/kubeedge/edge/test"
```

Comment:
@edisonxiang @sids-b @sujithsimon22 Could you please help check whether the test need import here? Thanks! :)

Reply:
The test should not be imported here at it is required only for integration test and e2e tests. It should be removed from the actual binary and should be add via code in the tests if required.
@edisonxiang @pavan187